austin/simulation.py

```diff
@@ -33,10 +34,13 @@ def cleanup_output_files():
     tracing.delete_output_files('prof')
 
 
-def run(run_list, injectables=None, warm_start=False):
+def run(run_list, injectables=None, warm_start=False, calibration = False):
```

Comment:
I see this `calibration` kwarg defined here but it's not exposed in L163 where the `run()` method is called. How does the user trigger calibration mode?